tests/integration/src/test/java/org/netxms/utilities/TestHelperForEpp.java

```diff
@@ -81,6 +81,44 @@ public static EventProcessingPolicyRule findOrCreateRule(final NXCSession sessio
       return testRule;
    }
    
+   /**
+    * Searching for test rule based on the specified comment, if not found, creates a new one.
+    * 
+    * @param session
+    * @param policy
+    * @param commentForSearching
+    * @param eventTemplate
+    * @param node
+    * @return EventProcessingPolicyRule
+    * @throws Exception
+    */
+   public static EventProcessingPolicyRule findOrCreateRuleWithoutNode(final NXCSession session, EventProcessingPolicy policy, String commentForSearching, EventTemplate eventTemplate)
```

Comment:
I think you can use same method as with node, but to node filed just give null or 0